docs/src/pages/demos/paper/PaperSheet.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles, Theme, createStyles, WithStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+
+const styles = (theme: Theme) =>
```

Comment:
Could you use the hook API?

Reply:
`useTheme`? is there a good example somewhere in other docs?

Reply:
If this is an existing demo it should result in the same behavior.

Reply:
@vitkon You can try https://github.com/mui-org/material-ui/pull/14947.

Reply:
ok, got it

Reply:
@oliviertassinari pushed, how does that look?

Reply:
Don't use hooks until #14947 is merged. We should adjust the ts files when we change the JS.